android/app/src/main/AndroidManifest.xml

```diff
@@ -9,14 +9,14 @@
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
 
     <application
-      android:name=".MainApplication"
+      android:name="com.textile.MainApplication"
```

Comment:
these were influencing app store automatic things, so wanted to reduce 'textilephotos' before anything went live